src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -96,7 +94,7 @@ private void execute(ProcBuilder builder) throws KaitaiException {
 		}
 	}
 
-	public Path generate(Log log) throws KaitaiException {
+	public Path generate(Logger log) throws KaitaiException {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' has incorrect indentation level 8, expected level should be 2\.